util/find-doc-nits

```diff
@@ -575,7 +580,7 @@ sub checkflags {
     }
     if ( scalar @undocced > 0 ) {
         foreach ( @undocced ) {
-            err("doc/man1/$cmd.pod: Missing -$_");
+            err("$doc: undocumented option -$_");
```

Comment:
$doc? Not $cmd?

Reply:
Yes, because it is an error in the documentation.  The old string was wrong because it didn't catch the openssl-foo style.